<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimelinePanel</class>
 <widget class="QWidget" name="TimelinePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>96</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Timeline</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="ui_verticalLayout">
     <item>
      <widget class="QScrollArea" name="ui_scrollArea">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <widget class="QWidget" name="ui_scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>554</width>
          <height>62</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ui_horizontalLayout">
       <item>
        <widget class="QSpinBox" name="ui_startFrameSpinBox">
         <property name="toolTip">
          <string>Start Frame</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ui_inFrameSpinBox">
         <property name="toolTip">
          <string>In Frame</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ui_outFrameSpinBox">
         <property name="toolTip">
          <string>Out Frame</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ui_endFrameSpinBox">
         <property name="toolTip">
          <string>End Frame</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="ui_fpsSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="ui_playLabel">
         <property name="text">
          <string>Play every </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ui_frameStepSpinBox">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string>th</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ui_frameLabel">
         <property name="text">
          <string> frame at </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ui_fpsSpinBox">
         <property name="toolTip">
          <string>Frames per Second</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string extracomment="Frames per Second"> fps</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>250</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ui_dotLabel">
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="ui_horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="ui_autoScrollAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/autoScrollIcon</normaloff>:/autoScrollIcon</iconset>
   </property>
   <property name="text">
    <string>Automatic Scrolling</string>
   </property>
   <property name="iconText">
    <string>Auto Scroll</string>
   </property>
   <property name="toolTip">
    <string>Automatic Scrolling [A]</string>
   </property>
  </action>
  <action name="ui_scaleToFitAction">
   <property name="icon">
    <iconset>
     <normaloff>:/scaleToFitIcon</normaloff>:/scaleToFitIcon</iconset>
   </property>
   <property name="text">
    <string>Scale to Fit</string>
   </property>
   <property name="toolTip">
    <string>Scale to Fit [F]</string>
   </property>
  </action>
  <action name="ui_realtimeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/realtimeIcon</normaloff>:/realtimeIcon</iconset>
   </property>
   <property name="text">
    <string>Realtime Mode</string>
   </property>
   <property name="toolTip">
    <string>Realtime Mode [R]</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
